src/config.ts

```diff
@@ -11,12 +11,16 @@ import { defaultTo } from 'lodash';
 import * as os from 'os';
 
 const DEFAULT_REDIS_PORT = 6379;
-const DEFAULT_EXPRESS_PORT = 8081;
+const DEFAULT_EXPRESS_PORT = '8081';
```

Comment:
I don't think you need to change this if you tweak the ordering below.
```suggestion
const DEFAULT_EXPRESS_PORT = 8081;
```

Reply:
Yes, indeed you are correct. 

I was a bit worried about the backward compatibility in it, by doing it the way you are suggesting the presence of a  PORT env var would break the existing application.

By doing it the suggested way, EXPRESS_PORT would have preceding over just PORT and making it work more like in the documentation. 

But this is a minimal issue; if you still think it's better your way,  I'm more then happy to adjust the code as you suggest.

Reply:
Hey yeah you're right good spot, my mistake, I'd meant to pass the arguments the other way around to `defaultTo`. I'll correct that suggestion now.

Reply:
Updated, if you're happy with those suggestions, I think we're probably good to get both PRs merged today 👍